client/src/editor/filters/mosaic-madness.tsx

```diff
@@ -0,0 +1,134 @@
+
+import * as React from 'react';
+import {  BaseFilter } from "./base-filter";
+
+export const MosaicMadnessFilter: BaseFilter = {
+    name: "Mosaic Madness",
+
+    Options: ({ setOptions }: { setOptions: any }) => {
+        React.useEffect(() => {
+            // Set default options when the component mounts
+            setOptions({ k: 5, tileSize: 10 });
+        }, [setOptions]);
+    
+        const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+            const { name, value } = event.target;
+            setOptions((prevOptions: any) => ({
+                ...prevOptions,
+                [name]: parseInt(value, 10),
+            }));
+        };
+        
+        return (
+            <div>
+                <label>
+                    Cluster Count (k):
+                    <input
+                        type="number"
+                        name="k"
+                        onChange={handleInputChange}
+                        defaultValue={5}
+                        min={1}
+                        max={50}
+                    />
```

Comment:
Use Joy UI. Take a look at Gaussian filter options. You can use Slider component to set a value